static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -356,3 +358,20 @@ export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
   }
   return false;
 }
+/**
+ * Converts the disease parent data from a flat array to a tree array for tree visualization
+ * @param data
+ * @returns an array consisting of disease name and a sub-array of its children
+ */
+export function formatDiseaseParentTreeData(
+  data: DiseaseParent[]
+): DiseaseParentTree[] {
```

Comment:
Looks like you're only returning one node, not an array?

Reply:
Good catch! Fixed it!